deny.toml

```
@@ -98,6 +98,10 @@ skip-tree = [
 
     # wait until zcash_client_backend update rustix
     { name = "rustix", version = "=0.38.44" },
+
+    # wait for windows-related updates
+    { name = "windows-strings", version = "=0.4.0" },
+    { name = "windows-core", version = "=0.61.0" }
```

```suggestion
    # wait for reqwest to update windows-registry
    { name = "windows-strings", version = "=0.3.1" }
```

Thanks for adding this, i am not totally sure why they are happening here but there are a few things to consider anyway:

- `windows-core` is not needed as the issue is just with `windows-strings`.
- We add the older version of the duplicated dependency here, in this case `0.3.1` of `windows-strings`.
- The comment tries to document what dependency is causing the issue, in this case this is `reqwest`.

---

I can update but just note: in my local testing windows-core also came up as an issue.